src/utils/cas/README

```diff
@@ -2,7 +2,7 @@ C AMULE STATS
 
 
 
-This is a little app that i've been doing cause i really didnt liked aStats.
+This is a little app that i've been using because I really didn't like aStats.
```

Comment:
this changes the meaning of the sentence, why not `doing` instead `using`?

Reply:
Oh I see, you meant it as in: `building` || `creating` || `hacking` || `coding` ?

Reply:
no problem, squashed and merged